assets/scss/app.scss

```diff
@@ -17,6 +18,9 @@ html {
 }
 
 // Bootstrap overrides
+.btn-twitter img:hover {
+  background-color: darken($brand-primary, 10%);
+}
```

Comment:
I added this since the in this with all other buttons the <a href= ></a> is the button that will be hovered upon. But with the twitter button the image is the button and I want to catch when the image is hovered and not when the wrapping <a href=""><img ...></a> is hovered. That comes from the way that the twitter button comes as a button not as a logo.